AirportDepartures.playground/Contents.swift

```diff
@@ -16,9 +16,101 @@ import UIKit
 //: e. Use a `String?` for the Terminal, since it may not be set yet (i.e.: waiting to arrive on time)
 //:
 //: f. Use a class to represent a `DepartureBoard` with a list of departure flights, and the current airport
-
-
-
+enum FlightStatus: String {
+    case boarding
+    case cancelled
+    case scheduled
+    case scheduledDelayed
+    case scheduledOnTime
+    case enRouteDelayed
+    case enRouteOnTime
+}
+
+enum AirportType {
+    case destination
+    case arrival
+}
+
+enum Airline {
+    case jetBlue
+    case virgin
+    case quantas
+}
+
+struct Airport {
+    let airportType: AirportType
+}
+
+struct Flight {
+    let destination: String
+    let airline: Airline
+    let departureTime: Date?
+    let terminal: String?
+    let status: FlightStatus
+}
+
+func unwrapDate(dateToUnwrap: Optional<Date>) -> String {
```

Comment:
I like the use of these unwrapping functions for the date and terminal. It makes your later code much cleaner when you need to access the flight's date and terminal.